lib/zendesk_apps_support/manifest.rb

```diff
@@ -101,6 +103,15 @@ def set_locations_and_hosts
         end
     end
 
+    def set_framework_options
+      @framework_options ||= {
+        noTemplate: no_template_locations,
+        singleInstall: single_install?,
```

Comment:
I'm not sure `singleInstall` is a framework option, seems more like a marketplace option.